ingenialink/dictionary.py

```diff
@@ -1040,6 +1082,36 @@ def __read_range(
             return range_min, range_max
         return None, None
 
+    def __read_monitoring(
+        self, pdo_access: RegCyclicType, monitoring_elem: Optional[ElementTree.Element]
+    ) -> Union[tuple[None, None, None], tuple[int, int, RegCyclicType]]:
+        """Process Monitoring element.
+
+        Args:
+            pdo_access: pdo access.
+            monitoring_elem: Monitoring element.
+
+        Returns:
+            Tuple with address, subnode and cyclic access
+            or None if the register is not monitoreable.
+
+        Raises:
+            ValueError: if register is PDO mappable but no monitoring entry is present.
+            ValueError: if register is not PDO mappable, and monitoring entry is present.
```

Comment:
I don't understand why we want this. The purpose of splitting pdo_mappable and monitoring was to decouple.
So there are no rules between them